test/helpers/gas-usage.ts

```diff
@@ -35,23 +35,23 @@ const MAX_GAS_PER_GROUP: any = {
   [GasGroup.REMOVE_OPERATOR]: 62600,
   [GasGroup.REMOVE_OPERATOR_WITH_WITHDRAW]: 62000,
 
-  [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 206000,
-  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 224900,
-  [GasGroup.REGISTER_VALIDATOR_WITHOUT_DEPOSIT]: 185500,
+  [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 228800,
```

Comment:
The gas raising seems to be due to function:
_registerValidatorPublicKey

Reply:
Maybe also due to mix up of commits in the PR

Reply:
mistery solved:
**main** branch
`struct Validator {
        address owner; -> 20 bytes
        bool active; -> 1 byte
    }`
less than 32 bytes, can be packed into 1 storage slot -> 20k gas 
 
 **SIP** branch
 `struct Validator {
        bytes32 hashedOperatorIds; -> 32 bytes
        bool active; -> 1 byte
    }`
more than 32 bytes, can not be packed into 1 storage slot -> 40k gas
@lior-blox @andrew-blox